### ISSUE TITLE
Crash when last account is deleted

### ISSUE THREAD

**Issue by przybylski** [nextcloud/android#117]
### Actual behaviour
- App crashes when last account is removed

### Expected behaviour
- Account setup dialog should appear
 
### Steps to reproduce
1. Setup only account
2. Delete account
3. Press back

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

PR: #132

**Comment by tobiasKaminsky** [comment]
This is a must have for the next release!

**Comment by AndyScherzinger** [comment]
I second that!

**Comment by przybylski** [comment]
Ill have a look than over the weekend.

**Comment by tobiasKaminsky** [comment]
PR: #132

**Comment by AndyScherzinger** [comment]
Tested and being included in the upcoming 1.2.0
